cdap-distributions/src/packer/scripts/lxde.sh

```diff
@@ -78,10 +78,34 @@ Icon=cdap
 Categories=GNOME;GTK;Development;
 EOF
 
+# CDAP SDK Menu Entry
```

Comment:
There are others in this file that are also marked as "Menu Entry" when they are a "Desktop Entry"...